src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const isObject = Object.entries(data[key]).length > 0;
```

Comment:
```suggestion
    const isObject = Object.entries(data[key]).length;
```
Любая цифра > 0 и так преобразуется в true